### ISSUE TITLE
`sparkctl` is not working anymore

### ISSUE THREAD

**Issue by kodelint** [GoogleCloudPlatform/spark-on-k8s-operator#679]
I have followed all the steps but `sparkctl` is not working anymore:
```
./sparkctl list -n spark
failed to list SparkApplications: the server could not find the requested resource (get sparkapplications.sparkoperator.k8s.io)
```

Here is what I followed:

```
dep ensure
go get -u k8s.io/code-generator/cmd/client-gen
go get -u k8s.io/code-generator/cmd/deepcopy-gen
go get -u k8s.io/code-generator/cmd/defaulter-gen
go get -u sigs.k8s.io/controller-tools/cmd/controller-gen
```
move to `sparkctl` folder and ran `go build -o sparkctl`.

But when I do this:

```
kubectl describe sparkapplication pyspark-pi
```

I get the response back.

Here is my `YAML`

```
apiVersion: "sparkoperator.k8s.io/v1beta2"
kind: SparkApplication
metadata:
  name: pyspark-pi
  namespace: spark
spec:
  type: Python
  pythonVersion: "2"
  mode: cluster
  image: "gcr.io/spark-operator/spark-py:v2.4.4"
  imagePullPolicy: Always
  mainApplicationFile: local:///opt/spark/examples/src/main/python/pi.py
  sparkVersion: "2.4.4"
  restartPolicy:
    type: OnFailure
    onFailureRetries: 3
    onFailureRetryInterval: 10
    onSubmissionFailureRetries: 5
    onSubmissionFailureRetryInterval: 20
  driver:
    cores: 1
    coreLimit: "1200m"
    memory: "512m"
    labels:
      version: 2.4.4
    serviceAccount: sparkoperator
  executor:
    cores: 1
    instances: 1
    memory: "512m"
    labels:
      version: 2.4.4
```

**Comment by araji** [comment]
same thing here .
change sparkctl to use betav2  solved it :
func doList(crdClientset crdclientset.Interface) error {
-       apps, err := crdClientset.SparkoperatorV1beta1().SparkApplications(Namespace).List(metav1.ListOptions{})
+       apps, err := crdClientset.SparkoperatorV1beta2().SparkApplications(Namespace).List(metav1.ListOptions{})
        if err != nil {
                return err
        }

**Comment by liyinan926** [comment]
Fixed in #681.

**Comment by kodelint** [comment]
Thanks @liyinan926
